gsoc-2024/work-product/Maria-Pana-Multiboot2-Support-in-Unikraft/Maria-Pana-Multiboot2.md

```diff
@@ -0,0 +1,148 @@
+# Multiboot2 support in Unikraft
+
+<img width="100px" src="https://summerofcode.withgoogle.com/assets/media/gsoc-generic-badge.svg" align="right" />
+
+## Project Overview
+
```

Comment:
Have a short phrase summarizing what the project does: add Multiboot2 support for Unikraft.